tests/.jshintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Hmm... how'd these files get created? They're supposed to be at the root of the repo.